### DISCUSSION TITLE
How can I programatically determine the image version?

### DISCUSSION THREAD

**Discussion by dundargoc** [actions/virtual-environments#7661] [General questions] [answered]
I'm specifically referring to the step "Set up job" in "Virtual Environment" -> "Version". I'd like to use this value in my workflow but I can't find a way to access this information.

**Comment by vpolikarpov-akvelon** [comment, marked as answer]
Hi, @dundargoc. There are environment variables `$ImageOS` and `$ImageVersion` available in all types of runners hosted by GitHub. Is it what you are looking for?

**Reply by dundargoc** [reply]
`$ImageVersion` is the one. Thanks!
